pages/regions.js

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, H6, Icon } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const cta = router.query.redirect;
+
+  return (
+    <section className="regions">
+      {data.regions.map((region) => {
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={cta === 'login' ? region.url : region.altUrl}>
+                <a>
+                  <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+                </a>
+              </Link>
+            </div>
+            <div className="regions__title">
+              <H6>
```

Comment:
We should be using the `<P>` jambonz-ui component here for correct type styling. The region titles need to be colored the same as their icon. The link text in this case should be the dark copy color and just the external link icon should be the jambonz pink color. The design is attached to the issue for reference.